app/models/compute_resource.rb

```diff
@@ -132,7 +132,7 @@ def host_compute_attrs(host)
 
   def host_interfaces_attrs(host)
     host.interfaces.select(&:physical?).each.with_index.reduce({}) do |hash, (nic, index)|
-      hash.merge(index.to_s => nic.compute_attributes)
+      hash.merge(index.to_s => nic)
```

Comment:
@domcleal thanks, does the following looks better to you?

`hash.merge(index.to_s => nic.compute_attributes)`
`hash.merge(index.to_s => { :ip => nic[:ip], **(nic[:ip6] ? {:ip6 => nic[:ip6]} : {}) })`

If not, where would be the best place to merge the IP attributes?